src/ui/Combobox/ComboboxValues.ts

```diff
@@ -54,13 +55,27 @@ export class ComboboxValues implements IComboboxValues {
         { id: `${this.combobox.id}-value-${index}`, className: 'coveo-combobox-value', role: 'option', tabindex: 0 },
         value.element
       ).el;
+      this.highlightCurrentQueryInSearchResults(value.element);
       value.element = elementWrapper;
       fragment.appendChild(value.element);
     });
 
     this.element.appendChild(fragment);
   }
 
+  private highlightCurrentQueryInSearchResults(searchResult: HTMLElement) {
+    if (this.combobox.options.highlightValueClassName) {
+      const regex = new RegExp('(' + this.combobox.element.querySelector('input').value + ')', 'ig');
+      let result = $$(searchResult).hasClass(this.combobox.options.highlightValueClassName)
```

Comment:
I think you meant to use a `const` here.

Reply:
You're right. I used `let` because I was modifying the `innerHTML` property, but I just read that you can modify the properties of a `const` obj. Didn't know that 😄